tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -497,21 +497,28 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                 if os.name != 'nt':
                     command = shlex.split(command)
                 else:
+                    orig_command = command
+                    command = shlex.split(command, posix=False)  # use non-posix method on Windows
```

Comment:
Please keep the variable name as `cl` as it was before the patch to avoid unnecessary renaming.

Reply:
I pushed another change to drop more renaming.